### PR TITLE
Feature/wf 574 c header links

### PR DESCRIPTION
**Pull request by adrianburtscher**
## Pull request
Adds c-header-links component
 
### Ticket
[WF-574](https://winterhalter-fenner.atlassian.net/browse/WF-574)
 
### Browser testing
- [http://localhost:8080/styleguide/product-detail](http://localhost:8080/styleguide/product-detail)
 
### Checklist
- [x] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [ ] Did run automated tests and linters
- [ ] Did review code and documentation
- [ ] Tested all links in project relevant browsers
- [ ] Tested all links in different screen sizes
- [ ] Re-assign ticket to developer

### CODE REVIEW

File: app/components/c-header-links.vue [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    props: {
+      isAccountActive: {
+        type: Boolean,
+        default: false,
+      },
+      isWishlistActive: {
+        type: Boolean,
+        default: false,
+      },
+      isLoginActive: {
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Rename to login, "is" Prefix is usually associated with getters
[ ] Add documentation

File: app/components/c-header-links.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        />
+        <span :class="b('label')">{{ $t('c-header-links.labelLogin') }}</span>
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+  import eIcon from '@/components/e-icon';
+
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: You can add basic documentation of a component here which shows up in the styleguide as description. Please explain how the component works and how it decides to show which icons.

File: app/components/c-linklist.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+### Styles
+
+#### Linklist
```
Comment (geroldboehler) [inline app/components/c-linklist.md]: [ ] Rename to default

File: app/components/c-linklist.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+```
+<c-linklist :items="[{ key: 'Montageanleitung als PDF', value: 'LinkTo: Montageanleitung als PDF' }, { key: 'Produktdatenblatt', value: 'LinkTo: Produktdatenblatt' }, { key: 'Zertifikate', value: 'LinkTo: Zertifikate' }]"/>
+```
```
Comment (geroldboehler) [inline app/components/c-linklist.md]: [ ] Add examples for all 3 props

File: app/components/c-linklist.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+### Styles
+
+#### Linklist
+```
+<c-linklist :items="[{ key: 'Montageanleitung als PDF', value: 'LinkTo: Montageanleitung als PDF' }, { key: 'Produktdatenblatt', value: 'LinkTo: Produktdatenblatt' }, { key: 'Zertifikate', value: 'LinkTo: Zertifikate' }]"/>
```
Comment (geroldboehler) [inline app/components/c-linklist.md]: [ ] Add a real link (e.g. https://www.google.com)
[ ] Set target="_blank"

File: app/components/c-header-links.vue [hunk 6]
```diff
@@ -0,0 +1,153 @@
+<template>
+  <div :class="b()">
+    <div v-if="userLoggedIn" :class="b('item', { active: accountActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkAccount')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-account"
+          width="25px"
+          height="25px"
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Move width and height to css

File: app/components/c-header-links.vue [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+      </a>
+    </div>
+    <div v-if="userLoggedIn" :class="b('item', { active: wishlistActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkWishlist')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-wishlist"
+          width="25px"
+          height="25px"
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Move width and height to css

File: app/components/c-header-links.vue [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+      </a>
+    </div>
+    <div v-if="!userLoggedIn" :class="b('item', { active: loginActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkLogin')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-account"
+          width="25px"
+          height="25px"
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Move width and height to css

File: app/components/c-header-links.vue [hunk 9]
```diff
@@ -0,0 +1,153 @@
+<template>
+  <div :class="b()">
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Please build the icons by using a for loop, code is mostly redundant

File: app/components/c-header-links.vue [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+      flex-direction: column;
+      align-items: center;
+      border-bottom: none;
+      padding: $spacing--10 $spacing--10 $spacing--5 $spacing--10;
+
+      @include media(xs) {
+        padding: $spacing--10 $spacing--20 $spacing--5 $spacing--20;
+      }
+
+      &:hover {
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Icons should probably be yellow on :hover. Please also implement :active and :focus according to https://projects.invisionapp.com/d/main#/console/13587539/289062420/preview. If they are not defined there, please ask Christoph Hirschbühl via Slack

File: app/components/c-header-links.vue [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+        border-bottom: none;
+      }
+    }
+
+    &__label {
+      @include font($font-size--10, 13, $font-weight--semi-bold);
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] 13px

File: app/translations/de.json [hunk 12]
```diff
@@ -20,6 +20,15 @@
     "xingTitle": "Xing"
   },
   "c-product-detail": {
-    "productDescriptionTitle": "Produktbeschreibung"
+    "productDescriptionTitle": "Produktbeschreibung",
+    "productDocumentsTitle": "Dokumente",
+    "productVideosTitle": "Videos"
+  },
+  "c-header-links": {
```
Comment (geroldboehler) [inline app/translations/de.json]: [ ] Rename to urls

File: app/components/c-header-links.vue [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+          inline
+          icon="i-wishlist"
+          width="25px"
+          height="25px"
+        />
+        <span :class="b('label')">{{ $t('c-header-links.labelWishlist') }}</span>
+      </a>
+    </div>
+    <div v-if="!userLoggedIn" :class="b('item', { active: loginActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkLogin')">
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Emit an event using this.$emit and handle redirection in c-header component

File: app/components/c-header-links.vue [hunk 14]
```diff
@@ -0,0 +1,153 @@
...
+          inline
+          icon="i-account"
+          width="25px"
+          height="25px"
+        />
+        <span :class="b('label')">{{ username }}</span>
+      </a>
+    </div>
+    <div v-if="userLoggedIn" :class="b('item', { active: wishlistActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkWishlist')">
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Emit an event using this.$emit and handle redirection in c-header component

File: app/components/c-header-links.vue [hunk 15]
```diff
@@ -0,0 +1,153 @@
+<template>
+  <div :class="b()">
+    <div v-if="userLoggedIn" :class="b('item', { active: accountActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkAccount')">
```
Comment (geroldboehler) [inline app/components/c-header-links.vue]: [ ] Emit an event using this.$emit and handle redirection in c-header component

File: package.json [hunk 16]
```diff
@@ -75,6 +75,7 @@
     "axios": "~0.18.0",
     "babel-polyfill": "~6.26.0",
     "bootstrap": "~4.0.0",
+    "eslint-plugin-brackets": "~0.1.3",
```
Comment (geroldboehler) [inline package.json]: [ ] Never add development packages to production dependencies

File: app/components/c-linklist.md (hunk 4 above)
Comment (adrianburtscher) [reply app/components/c-linklist.md]: Do you mean links to actually existing documents?

**Pull request closed without merging**